collectors/fda_dap/spider.py

```diff
@@ -264,6 +270,38 @@ def _get_document_name(el):
                 if name:
                     return name
 
+        def _fix_multipart_review_documents(item, default_document_name='Review'):
+            '''Fixes DAP pages with documents split in many parts.
+
+            If this is the case of the current page, this method will merge the
+            multipart documents in a single one with the name of the document
+            minus the "Part X", or the "default_document_name".
+            '''
+            partial_documents = [doc for doc in item['documents']
+                                 if re.search(r'Part \d+', doc['name'])]
+            keys = set([re.sub(r'Part \d+', '', doc['name']).strip()
```

Comment:
There can be in the future, but I couldn't find any by going over our scrapped results on re:dash